lib/core/presentation/widgets/event/rsvp_event_success_popup_widget.dart

```diff
@@ -0,0 +1,169 @@
+
+class RSVPEventSuccessPopupWidget extends StatelessWidget {
+  const RSVPEventSuccessPopupWidget({
+    super.key,
+    required this.event,
+  });
+
+  final Event event;
+
+  @override
+  Widget build(BuildContext context) {
+    final colorScheme = Theme.of(context).colorScheme;
+    final t = Translations.of(context);
+    return Scaffold(
+      backgroundColor: colorScheme.primary,
+      body: SafeArea(
+        child: Stack(
+          children: [
+            Column(
+              mainAxisSize: MainAxisSize.max,
+              children: [
+                const _SuccessCircle(),
+                SizedBox(
+                  height: 56.w,
+                ),
+                Padding(
+                  padding: EdgeInsets.symmetric(horizontal: Spacing.xLarge),
+                  child: Text(
+                    t.event.youreIn,
+                    style: Typo.extraLarge.copyWith(
+                      fontFamily: FontFamily.nohemiVariable,
+                      fontWeight: FontWeight.w900,
+                      height: 1.2,
+                    ),
+                  ),
+                ),
+                SizedBox(height: Spacing.superExtraSmall),
+                Padding(
+                  padding: EdgeInsets.symmetric(horizontal: Spacing.xLarge),
+                  child: Text(
+                    t.event.rsvpSuccessful(eventName: event.title ?? ''),
+                    style: Typo.mediumPlus.copyWith(
+                      fontWeight: FontWeight.w400,
+                      color: colorScheme.onSecondary,
+                    ),
+                    textAlign: TextAlign.center,
+                  ),
+                ),
+                SizedBox(height: Spacing.xLarge),
+                Padding(
+                  padding: EdgeInsets.symmetric(horizontal: Spacing.xLarge),
+                  child: LinearGradientButton(
+                    mode: GradientButtonMode.lavenderMode,
+                    height: Sizing.large,
+                    textStyle: Typo.medium.copyWith(
+                      fontWeight: FontWeight.w600,
+                      color: colorScheme.onPrimary.withOpacity(0.87),
+                      fontFamily: FontFamily.nohemiVariable,
+                    ),
+                    radius: BorderRadius.circular(LemonRadius.small * 2),
+                    label: t.event.takeMeToEvent,
+                  ),
+                )
+              ],
+            ),
+            Positioned(
+              top: 0,
+              left: Spacing.smMedium,
+              height: Sizing.xLarge,
+              child: LemonBackButton(
+                color: colorScheme.onPrimary,
+              ),
+            ),
+            Positioned(
+              top: Spacing.smMedium / 2,
+              right: Spacing.smMedium,
+              child: Container(
+                width: 125.w,
+                height: Sizing.medium,
+                decoration: BoxDecoration(
```

Comment:
Better use padding instead of set fixed width/height. This might cause widget overflow error

Reply:
I dont think 125.w would overflow on any device ? :/

Reply:
fair enough